modal_proto/api.proto

```diff
@@ -111,6 +111,7 @@ enum DataFormat {
   DATA_FORMAT_PICKLE = 1; // Cloudpickle
   DATA_FORMAT_ASGI = 2; // "Asgi" protobuf message
   DATA_FORMAT_GENERATOR_DONE = 3; // "GeneratorDone" protobuf message
+  DATA_FORMAT_CBOR = 4;
```

Comment:
Is it okay to include proto changes in the same PR as the implementation changes or should proto changes be in their own PR?

Reply:
I don't think it's necessary in this case, but normally good practice yeah :)